docs/ListItemTag.md

```diff
@@ -24,8 +24,9 @@ import { ListItemTag } from '@brightlayer-ui/react-native-components';
 | --------------- | ----------------------------- | ----------------------------- | -------- | ------------------------- |
 | backgroundColor | Background color of the label | `string`                      | no       | `theme.colors.primary`    |
 | fontColor       | Text color for the label      | `string`                      | no       | Varies based on the theme |
+| fontSize        | Size of the text              | `number`                      | no       | 10                        |
 | label           | The label text                | `string`                      | yes      |                           |
-| theme           | Theme value overrides         | `$DeepPartial<ExtendedTheme>` | no       |                           |
+| theme           | Theme value overrides         | `$DeepPartial<ExtendedTheme>` | no       | `theme.colors.onPrimary`  |
```

Comment:
why is this the default theme value?

Reply:
These changes were done before our discussion. I have removed and fixed the default theme value [here](https://github.com/etn-ccis/blui-react-native-component-library/pull/482/files#diff-81c387f5a87d21c14fd9236048d5c046b799836a3fe677fa2906d34e4dd6049fR65)